modin/engines/ray/pandas_on_ray/frame/partition.py

```diff
@@ -13,15 +13,19 @@
 
 import pandas
 
-from modin.engines.base.frame.partition import BaseFramePartition
+from modin.config import EnablePartitionIPs
 from modin.data_management.utils import length_fn_pandas, width_fn_pandas
+from modin.engines.base.frame.partition import BaseFramePartition
 from modin.engines.ray.utils import handle_ray_task_error
 
 import ray
-from ray.services import get_node_ip_address
 from ray.worker import RayTaskError
 
 
+if EnablePartitionIPs.get():
+    from ray.services import get_node_ip_address
```

Comment:
```suggestion
from ray.services import get_node_ip_address
```
same note about config variable being changed at runtime